public/css/style.css

```diff
@@ -0,0 +1,32 @@
+* {
```

Comment:
style.css is too generic a name - think of something that explains what you are doing

Reply:
Done.